rust/moving-average/src/lib.rs

```diff
@@ -15,7 +15,7 @@ pub struct MovingAverage {
 
 impl MovingAverage {
     pub fn new(window_millis: u64) -> Self {
-        let now = chrono::Utc::now().naive_utc().timestamp_millis() as u64;
+        let now = chrono::Utc::now().naive_utc().and_utc().timestamp_millis() as u64;
```

Comment:
this looked weird to me, should it be `chrono::Utc::now().unwrap().timestamp_millis()`? (i don't write rust code)

Reply:
theres a deprecation warning :-(